### DISCUSSION TITLE
미안

### DISCUSSION THREAD

**Discussion by Nubula01** [RimWorldKorea/RMK#354] [질문]
이름만 걸어놓고 딴겜만 하다 지금 기억나서 왔다 미안하다

위키에 번역 가이드북 읽고 한번 추출해봤는데 번역 가이드북이랑 https://github.com/RimWorldKorea/RMK/discussions/331 여기서 예시로 보여주는 이미지랑 다르더라고 어느쪽에 맞춰야 하는건지 모르겠네

일단 실험삼아 추출해서 뽑아본건 가이드북하고 같게 나오더라

어.. 또 뭐 해야하는게 있으면 알려주면 고맙겠음

**Comment by RimWorldKorea** [comment]
월급 주는 것도 아닌데 의무감 느낄 필요는 없음

위키에서 말한대로 뽑은건 아마 시트가 하나에 F열까지밖에 데이터가 없을거고

언급한 예시에 있는건 xml파일밖에 없는 번역이나 번역본이 여러개인 번역을 xlsx 파일에서 합칠 때

데이터를 맞추기 위해 추가적인 작업을 한걸 보여주는거임
  

추가로 여유가 있으면 파일 업로드 하는 법 배워두면 좋음

아직 정리된 매뉴얼은 없긴 한데 나중에 쓰면 그거 봐도 되고

아니면 #339 참고해서 미리 하나씩 해봐도 되고

**Comment by Nubula01** [comment]
한림쪽에 있던거 따로 번역 업뎃한건 없지? 일단 그거부터 해야겠네 
링크 준거는 보니까 클라우드 비슷한건가 싶은데 함 해보겠음

**Reply by RimWorldKorea** [reply]
변경된 번역이 있었어도
그게 한림에서 넘어온 번역인지 아닌진 따로 기록을 안해서 모르겠네

체크하고 싶으면 코드 탭에서 검색하면
수정 이력 나오니깐 확인해보면 될 듯
